tools/ci_connector_ops/ci_connector_ops/qa_engine/inputs.py

```diff
@@ -33,18 +37,21 @@ def fetch_adoption_metrics_per_connector_version() -> pd.DataFrame:
     """Retrieve adoptions metrics for each connector version from our data warehouse.
 
     Returns:
-        pd.DataFrame: A Dataframe with adoption metrics per connector version.
+        pd.DataFrame: A dataframe with adoption metrics per connector version.
     """
-    # TODO: directly query BigQuery
-    # use query in https://airbyte.metabaseapp.com/question/1642-adoption-and-success-rate-per-connector-version-oss-cloud
-    return pd.DataFrame(columns=[
+    connector_adoption_sql = files("ci_connector_ops.qa_engine").joinpath("connector_adoption.sql").read_text()
+    bq_credentials = service_account.Credentials.from_service_account_info(json.loads(os.environ["QA_ENGINE_AIRBYTE_DATA_PROD_SA"]))
```

Comment:
Confirming this secret is already in Github?

Reply:
Yes it is.